node/chainSimulator/components/processComponents.go

```diff
@@ -155,12 +154,25 @@ func CreateProcessComponents(args ArgsProcessComponentsHolder) (*processComponen
 		return nil, err
 	}
 
-	whiteListRequest, err := disabled.NewDisabledWhiteListDataVerifier()
+	lruCache1, err := cache.NewLRUCache(100000)
+	if err != nil {
+		return nil, err
+
+	}
+	whiteListRequest, err := interceptors.NewWhiteListDataVerifier(lruCache1)
```

Comment:
whiteListHandler instead of whiteListRequest ?

Reply:
changed